pkg/proxy/winkernel/proxier.go

```diff
@@ -1111,15 +1130,11 @@ func (proxier *Proxier) syncProxyRules() {
 				svcInfo.targetPort = int(ep.port)
 			}
 
-			if len(ep.hnsID) > 0 {
-				newHnsEndpoint, err = hns.getEndpointByID(ep.hnsID)
-			}
-
 			if newHnsEndpoint == nil {
```

Comment:
this will always be `nil` now, can simplify to `newHnsEndpoint := queriedEndpoints[ep.IP()]` and remove `var newHnsEndpoint *endpointsInfo` from a few lines above

Reply:
Updated, thanks